[docs][filebeat]: add 4th step in migrate-to-filestream (#39863)
elastic/beats: filebeat/docs/howto/migrate-to-filestream.asciidoc

|backoff.max
|===

=== Step 4

> The events produced by `filestream` input conatains `take_over` tag.

Reviewer: ```suggestion
The events produced by `filestream` input contains `take_over` tag.
```

> You can filter on this tag in Kibana and see the events which came from a filestream in the "take over" mode. 

Reviewer: ```suggestion
You can filter on this tag in Kibana and see the events coming from a filestream in the "take_over" mode. 
```

> The events produced by `filestream` input with `take_over: true` contain `take_over` tag.
> You can filter on this tag in Kibana and see the events which came from a filestream in the "take_over" mode. 
> 
> Once you start receiving event with this tag, you can remove `take_over: true` and restart the fileinput again.

Reviewer 2: ```suggestion
The events produced by `filestream` input with `take_over: true` contain a `take_over` tag.
You can filter on this tag in Kibana and see the events which came from a filestream in the "take_over" mode. 

Once you start receiving events with this tag, you can remove `take_over: true` and restart the fileinput again.
```
